Specs/DateTime/Spanish/DatePeriodExtractor.json

```diff
@@ -582,28 +461,6 @@
       }
     ]
   },
-  {
-    "Input": "Estare afuera el verano",
```

Comment:
We could change this test to `Estare fuera durante el verano` instead of deleting it.

Reply:
Hi @enzocanoo 
Thanks a lot for your careful review to this PR.
We have make some changes according to your suggestions.
Besides, we still delete some of the specs which our Spanish intern think it's wrong expression.
As I don't have much knowledge about Spanish, could you please check the deleted specs to double check that they really should be deleted?

Thanks again!